[No issue] Intro tidy up (#229)
openmsupply/remote-server: docs/content/docs/getting-started/_index.md

toc = true
top = false
+++

## Welcome

> Thanks for visiting our developer documentation.

Reviewer: Who is the document "owner" here? Do we want to be a bit more inclusive (i.e. avoid possessive pronouns)? 

```suggestion
Thanks for visiting the Open mSupply developer documentation.
```

> If you're looking for "How to use mSupply" documentation, you're in the wrong place. For "old" mSupply they're [here](https://docs.msupply.org.nz/) and for Open mSupply they're not ready yet. Stay tuned.

Reviewer: This is a bit awkward to parse, maybe something like:

```suggestion
If you're looking for "How to use mSupply" documentation, you're in the wrong place. We're still working on the Open mSupply documentation (stay tuned!), but documentation for previous versions of mSupply can be found [here](https://docs.msupply.org.nz/)
```

Note also replacing "old" with "previous" (I think "old" implies "outdated", which isn't true as mSupply is still being maintained?).

## Contributing

> We realise you might read these docs for a number of reasons:

Reviewer: ```suggestion
We understand that you may be reading these docs for a number of reasons:
```

> * You might just be interested.
> * You might have forked mSupply and want to know how to modify it. 
> * You might be considering extending mSupply by writing a plugin.
> * Or you might be considering contributing, to the main code base.

Reviewer: Minor tidy up and re-ordering.

```suggestion
* You might be considering contributing to the mSupply project, or
* forking your own modified version of mSupply, or
* extending mSupply by writing a plugin, or
* you might just be interested in learning more about Open mSupply!
```

> Any of those reasons are welcome, and we hope these docs help you. Please make an issue for errors you find, and ideally make a branch & a PR to fix the error. Thanks!

Reviewer: Minor tidy up.

```suggestion
Any and all of the above reasons are welcome! We also welcome documentation contributions, so feel free to make an issue and pull request to fix any errors you find. We hope these docs help you!
```
